tests/bson/bson-fromPHP_error-008.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+BSON\fromPHP(): PHP documents with circular references
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+echo "\nTesting packed array with circular reference\n";
+
+echo throws(function() {
+    $document = ['x' => 1, 'y' => [1, 2, 3]];
```

Comment:
This looks redundant, since `bson-fromPHP_error-004.phpt` already tests `$document = ['x' => 1, 'y' => []];` and asserts that the field path is "y.0". All this is testing is that `$document['y'][] = &$document['y'];` on the subsequent line appends the fourth element in a packed array instead of the first.